Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/InputType.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Given the device nature of the use of this enum, I'll remove it from Input (or create a new one in _Core)